src/Http/GraphRequest.php

```diff
@@ -321,31 +321,35 @@ function ($reason) {
     */
     public function download($path, $client = null)
     {
+        //Add custom error handler to catch file I/O warnings
+        set_error_handler(function() {});
```

Comment:
I don't think it's a library's responsibility to decide whether I want to completely silence warnings/errors like this. When using a framework like Symfony or Laravel this will all be taken care of already. This is more or less a VB6-style `on error resume next` approach while we should always just want things to fail as hard as possible.

What case is this construct trying to solve?